temp_changelog

```diff
@@ -0,0 +1,6 @@
+## Fleet 4.59.1 (Nov 15, 2024)
```

Comment:
shouldn't be checked in (could even add to .gitignore to make this not happen with a `git add .` )